criu/config.c

```diff
@@ -587,8 +587,10 @@ int parse_options(int argc, char **argv, bool *usage_error,
 			opts.final_state = TASK_ALIVE;
 			break;
 		case 'x':
-			if (optarg && unix_sk_ids_parse(optarg) < 0)
+			if (optarg && unix_sk_ids_parse(optarg) < 0) {
+				pr_err("unix_sk_ids_parse failed.\n");
 				return 1;
```

Comment:
`unix_sk_ids_parse()` has informative error messages on the error paths. I don't think that adding "unix_sk_ids_parse failed" will improve user's understanding of what happened. Moreover, it may only confuse them ;-)

Reply:
Thanks for your patience and suggestions, @rppt 
I've updated the error messages to be more user-friendly. I'm open to feedback if the updated messaging is still not quite up-to-standard :)